zerver/lib/import_realm.py

```diff
@@ -861,6 +872,7 @@ def import_message_data(import_dir: Path) -> None:
 
         re_map_foreign_keys(data, 'zerver_message', 'id', related_table='message', id_field=True)
         bulk_import_model(data, Message)
+        fix_message_rendered_content(data, 'zerver_message')
```

Comment:
I think we want something like this.  The reason is that in production use, it's quite annoying to have the import crash here (because the tool isn't designed to be rerun without starting from the beginning), whereas it's only mildly annoying to have a couple messages that didn't markdown-render at first.

```
 
         re_map_foreign_keys(data, 'zerver_message', 'id', related_table='message', id_field=True)
         bulk_import_model(data, Message)
-        fix_message_rendered_content(data, 'zerver_message')
+        try:
+            fix_message_rendered_content(data, 'zerver_message')
+        except Exception as e:
+            logging.warning("Error in markdown rendering for message ID %s; continuing" % (<message_id>))
```

Needs some cleanup.  And then I think we can have a tool that tries to render any un-rendered messages that one can run afterwards.